src/jdk.compiler/share/classes/com/sun/tools/javac/util/Log.java

```diff
@@ -214,11 +213,6 @@ public enum WriterKind { NOTICE, WARNING, ERROR, STDOUT, STDERR }
      */
     public Set<String> expectDiagKeys;
 
-    /**
-     * Set to true if a compressed diagnostic is reported
-     */
-    public boolean compressedOutput;
```

Comment:
I see that this patch drops the option for compressing diagnostic - this seems to go a step too far, and I'm not sure this is what we want, at least in this patch. I believe there's an orthogonal discussion to be had there. I think the flag should stay as is for now (and  control  whether diagnostics should be rewritten at all in Log).